extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -96,9 +104,9 @@ public function match( string $type, $content ) {
 		if ( is_array( $content ) ) {
 			$content = array_unique( $content );
 		}
-		
+
 		$result = $this->sendToPhalanxDaemon( 'match', [ 'type' => $type, 'content' => $content ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "match" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)